src/internal/components/cartesian-chart/styles.scss

```diff
@@ -40,6 +40,20 @@
   fill: awsui.$color-text-body-secondary;
 }
 
+.ticks__text__group {
+  text-align: center;
+}
+
+.ticks__text__bottom {
```

Comment:
I'm not sure this is the clearest naming, perhaps using `categorical` instead of `bottom` might distinguish more obviously?